documentation/image_duplicates.md

```diff
@@ -0,0 +1,167 @@
+# Tutorial: Finding Image Duplicates using the ImageDuplicates Feature Extractor
+In this tutorial, we will explore how to utilize the ImageDuplicates class to identify duplicate images within a specified directory containing images. The ImageDuplicates class is a feature extractor that employs Difference Hashing to detect duplicate images based on their hash codes. We will guide you through the process of initializing and using this class to find duplicates within the COCO2017 detection dataset and examine the results.
+
+
+## Step 1: Adding the ImageDuplicates to the Image Features Report Section Features List
```

Comment:
To keep style constant with other `.md`, I think it's better to write `## 1. Adding ...` instead of `## Step1: Adding...`